### PR TITLE
Fix upload image

### PR DESCRIPTION
**Pull request by mckinsel**
- Clean up the imports so we don't have two paths
- Fix h5py str/bytes issue
- Fix path to R script
- Fix moved requirements.txt file

### CODE REVIEW

File: backend/processing_image/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3
+4
```
Comment (Bento007) [inline backend/processing_image/VERSION:1]: do we still need to manually update this?

File: backend/corpora/dataset_processing/process.py [hunk 2]
```diff
@@ -10,21 +10,11 @@
...
-
-except ImportError:
-    # We're in the container
-    sys.path.append("/code")
-    from common.entities.dataset import Dataset
-    from common.corpora_orm import DatasetArtifactFileType, DatasetArtifactType
-    from common.utils import dropbox
-    from common.utils.db_utils import db_session
-    from .download import download
+from backend.corpora.common.entities.dataset import Dataset
```
Comment (Bento007) [inline backend/corpora/dataset_processing/process.py:13]: can you update data_processing/download as well.

File: backend/processing_image/Makefile [hunk 3]
```diff
@@ -9,11 +9,11 @@ bump_version:
...
 build:
-	mkdir -p code/chalice/api_server
-	cp -R ../corpora/dataset_processing code/
-	cp -R ../corpora/common code/
-	cp -R ../requirements.txt code/
-	cp -R ../chalice/api_server/requirements.txt code/chalice/api_server/requirements.txt
+	mkdir -p code/backend/chalice/api_server
+	cp -R ../corpora code/backend
+	cp -R ../../requirements.txt code/
+	cp -R ../chalice/api_server/requirements.txt code/backend/chalice/api_server/requirements.txt
```
Comment (Bento007) [inline backend/processing_image/Makefile:15]: why do you need the api_server/requirements.txt?

Comment (mckinsel) [reply backend/processing_image/Makefile:15]: The main requirements.txt references it

File: backend/corpora/dataset_processing/process.py (hunk 2 above)
Comment (mckinsel) [reply backend/corpora/dataset_processing/process.py:13]: Yeah, updated

File: backend/processing_image/VERSION (hunk 1 above)
Comment (mckinsel) [reply backend/processing_image/VERSION:1]: Currently yeah

File: backend/corpora/dataset_processing/process.py [hunk 4]
```diff
@@ -208,8 +199,7 @@ def make_seurat(local_filename):
     """Create a Seurat rds file from the AnnData file."""
 
     seurat_proc = subprocess.run(
-        ["Rscript", os.path.join(os.path.abspath(os.path.dirname(__file__)), "make_seurat.R"), local_filename],
-        capture_output=True,
+        ["Rscript", "/code/backend/corpora/dataset_processing/make_seurat.R", local_filename], capture_output=True
```
Comment (jgadling) [inline backend/corpora/dataset_processing/process.py:202]: Can we use os.path.abspath here?

Comment (mckinsel) [reply backend/corpora/dataset_processing/process.py]: Yeah

File: backend/corpora/dataset_processing/process.py [hunk 5]
```diff
@@ -176,6 +166,7 @@ def _get_term_pairs(base_term):
         ]
 
     return {
+        "name": adata.uns["title"],
```
Comment (Bento007) [inline backend/corpora/dataset_processing/process.py:169]: add this to your list of changes

**Review by Bento007 — APPROVED** [review]
LGTM

**Pull request merged**